pallets/avn-proxy/src/tests/proxy_tests_with_fees.rs

```diff
@@ -420,7 +422,7 @@ mod charging_fees {
                         inner_call,
                         payment_authorisation
                     ),
-                    BalanceError::<TestRuntime, _>::InsufficientBalance
+                    TokenError::FundsUnavailable
```

Comment:
should this change? its an interface of the pallet.
Also what is BalanceError? where is it defined

Reply:
I am assuming this is coming from the Balances pallet. I think BalanceError is an alias for Error of the Balances pallet.

Reply:
this is because they have changed how they handle when you have 0 balance vs having some insufficient amount of a token. `BalanceError` is the alias for errors coming from the balances pallet.